src/api/getProductComments.js

```diff
@@ -0,0 +1,24 @@
+const getProductComments = async (productId, limit = 10, cursor = null) => {
+  try {
+    const params = new URLSearchParams();
+    params.append("limit", limit);
```

Comment:
💊 제안
URLSearchParams 함수를 사용하신 것 좋아요! 아래처럼 초기값으로 limit를 넘겨주셔도 됩니다.

```suggestion
    const params = new URLSearchParams({ limit });
```